jokes/index.json

```diff
@@ -2266,5 +2266,11 @@
     "type": "general",
     "setup": "My older brother always tore the last pages of my comic books, and never told me why.",
     "punchline": "I had to draw my own conclusions."
+  },
+  {
+    "id": 389,
+    "type": "programming",
+    "setup": "A male developer often gets called as a Dev, then what would you call a female developer?.",
```

Comment:
I think the `.` should be removed @Raj-Thombare

Reply:
Yeah that's the main cause